src/runtime/policy/policy.ts

```diff
@@ -279,6 +290,65 @@ export class PolicyField {
       label: label === 'raw' ? '' : label,
     };
   }
+
+  getRestrictedFields(parentType: Type) {
+    const field = parentType.getEntitySchema().fields[this.name];
+    return [this.name, this.restrictField(field)];
```

Comment:
returning an object here:

`return {name: this.name, field: this.restrictField(field)};`

Would make the reduce above easier to read.

Instead of: `.reduce((fields, field) => ({...fields, [field[0]]: field[1]}), {});`
We could do: `.reduce((fields, {name, field}) => ({...fields, name: field}), {});`

Reply:
Done.